docs/next.config.js

```diff
@@ -14,6 +14,17 @@ const nextConfig = {
 		NEXT_PUBLIC_GITHUB_PR_PREVIEW_NUMBER: process.env.GITHUB_PR_PREVIEW_NUMBER,
 		NEXT_PUBLIC_BASE_PATH: basePath,
 	},
+	experimental: {
```

Comment:
@dagoof We've had to add this because of the Main nav component now uses floating-ui under the hood (because of the dropdown menu).

We also do this in the example-form and example-site, so nothing new here.